units/src/result.rs

```diff
@@ -254,6 +254,18 @@ impl MathOp {
 
     /// Returns `true` if this operation error'ed due to division by zero.
     pub fn is_div_by_zero(self) -> bool { !self.is_overflow() }
+
+    /// Returns `true` if this operation error'ed due to addition.
+    pub fn is_addition(self) -> bool { self == MathOp::Add }
+
+    /// Returns `true` if this operation error'ed due to subtraction.
+    pub fn is_subtraction(self) -> bool { self == MathOp::Sub }
+
+    /// Returns `true` if this operation error'ed due to multiplication.
+    pub fn is_multiplication(self) -> bool { self == MathOp::Mul }
+
+    /// Returns `true` if this operation error'ed due to negation.
+    pub fn is_negation(self) -> bool { self == MathOp::Neg }
```

Comment:
Cool, yes, these are nice!

Though you're missing `is_division_by_zero` and the commit message would then need to say math error not overflow.

Reply:
> Though you're missing is_division_by_zero 

That already exists: https://github.com/rust-bitcoin/rust-bitcoin/blob/742f8f84a94705b8e00920e5df27cd18ed803ad5/units/src/result.rs#L256

> and the commit message would then need to say math error not overflow.

Sure, can update the commit message

Reply:
done